configs/Gigabyte/GA-B550M-AORUS-PRO.conf

```diff
@@ -0,0 +1,21 @@
+# Requires it87 driver with support for IT8688E Super I/O chip
+# Tested with https://github.com/frankcrawford/it87 on B550M AORUS PRO (rev. 1.0)
+
+chip "it8688-isa-0a40"
+
+    # Labeled as marked on board
+    label fan1 "CPU_FAN"
+    label fan2 "SYS_FAN1"
+    label fan3 "SYS_FAN2"
+    label fan4 "SYS_FAN3"
+    label fan5 "CPU_OPT"
+
+    # Labeled according to names in UEFI "Smart Fan" utility
+    label temp1 "System 1"
+    label temp2 "System 2"
```

Comment:
When temperature warning is disabled, min/max readings are broken:
```
(low  = +127.0°C, high = +127.0°C)
(low  =  +0.0°C, high = -125.0°C)
```

After enabling it everything matches values set in uefi except system {1, 2} which show:
```(low  = -128.0°C, high = +70.0°C)```

Should I set min values to 0 for them here?